tips/0070-localizations/index.md

```diff
@@ -0,0 +1,106 @@
+# Localizing Flutter apps with code gen
+
+Did you know that you can set up your Flutter app localizations in 5 minutes, using Flutter's tool generation? 🗺
+
+Let’s start by adding the required packages: `flutter_localizations` and `intl`.
```

Comment:
Minor nitpick: strictly speaking, I think `intl` is not necessary.

Any specific reason you included it here?

Reply:
It's not, but then the analyzer will leave a warning.

![Screen Shot 2022-09-25 at 12 50 44 PM](https://user-images.githubusercontent.com/41123719/192137692-5b7f8620-1075-43ec-b328-97bfe15e9d55.png)

Reply:
Ok got it